samples/README.md

```diff
@@ -1,53 +0,0 @@
-# Code samples for the Compute Engine library
```

Comment:
Could you add in a README here to let folks know that the samples have migrated:

```
Samples migrated
================

New location: https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/compute/client_library
```

Reply:
Sure, good idea :)